hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1701,6 +1704,18 @@ message DeleteSnapshotRequest {
   optional uint64 deletionTime = 4;
 }
 
+message SnapshotMoveDeletedKeysRequest {
+  optional SnapshotInfo fromSnapshot = 1;
+  optional SnapshotInfo nextSnapshot = 2;
```

Comment:
I think nextSnapshot is no longer used?

Reply:
It is being used in `OMSnapshotMoveDeletedKeysResponse`